tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -207,5 +217,25 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            var date = DateTime.Now.AddDays(-31);
+
+            foreach (var logFile in Directory.EnumerateFiles(logsDirectory, "dotnet-tracer-*"))
```

Comment:
You could be even more specific
- dotnet-tracer-managed-*-.log
- dotnet-native-loader-*.log
- DD-DotNet-Profiler-Native-*.log